src/main/scala/io/iohk/ethereum/domain/Blockchain.scala

```diff
@@ -151,6 +160,16 @@ class BlockchainImpl(
 
   override def getTotalDifficultyByHash(blockhash: ByteString): Option[BigInt] = totalDifficultyStorage.get(blockhash)
 
+  override def getAccount(address: Address, blockNumber: BigInt): Option[Account] =
+    getBlockHeaderByNumber(blockNumber).flatMap { bh =>
+      val mpt = new MerklePatriciaTrie[Address, Account](
```

Comment:
I thought that the `hashFn` hashes the address (what is it for then?). Good catch. The serializer in `Address` comp obj has been updated.